tools/sharkbait/src/sharkbait/permissions.clj

```diff
@@ -21,9 +22,10 @@
   "Sets the allowed actions for a permission definition."
   [permission-def]
   (let [delegate (.getAttributeDefActionDelegate permission-def)]
-    (.configureActionList delegate ["read" "write" "own"])
-    (add-implied-action delegate "own" "write")
-    (add-implied-action delegate "write" "read"))
+    (.configureActionList delegate [read write own originator])
+    (add-implied-action delegate originator own)
```

Comment:
OK, that makes more sense. It might have helped if I read the JIRA title for this issue first `Update the app category listing endpoints to include a Shared With Me category` :confounded: